torch/csrc/jit/runtime/register_special_ops.cpp

```diff
@@ -433,6 +436,15 @@ RegisterOperators reg({
         "aten::set_grad_enabled(bool val) -> ()",
         [](Stack* stack) { torch::GradMode::set_enabled(pop(stack).toBool()); },
         aliasAnalysisConservative()),
+    Operator(
+        "aten::cuda_getCurrentStream(int64_t val) -> intrusive_ptr",
```

Comment:
```suggestion
        "aten::cuda_getCurrentStream(int64_t val) -> __torch__.torch.cuda.Stream",
```

`torch/csrc/jit/runtime/register_distributed_ops.cpp` has an example of something similar to what we want to do: 

Custom class registration:
```
static auto workerInfo =
    torch::class_<dist_rpc::WorkerInfo>("dist_rpc", "WorkerInfo")
        .def(torch::init<std::string, int64_t>());
```

Custom op registration:
```
     Operator(
         "aten::owner(RRef(t) self) -> __torch__.torch.classes.dist_rpc.WorkerInfo",
         [](Stack* stack) {
           auto rref = pop(stack).toRRef();
           push(
               stack,
               torch::make_custom_class<distributed::rpc::WorkerInfo>(
                   rref->ownerName(), rref->owner()));
         },
         aliasAnalysisFromSchema()),
```

I'm not sure about the difference between using `TORCH_LIBRARY` (from the custom class tutorial) and `torch::class_` directly.

Reply:
Oh ok , cool. This helps. Let me try this out. Thanks.